### PR TITLE
Many documentation changes + other small changes

### PR DESCRIPTION
**Pull request by theomgifford**
Details in commit descriptions.

### CODE REVIEW

File: bondpad.py [hunk 1]
```diff
@@ -34,6 +40,7 @@ class Bondpad(Component):
     _defaults['taper_length'] = 300
     _defaults['launcher_padding'] = 167.44
     _defaults['bond_pad_length'] = 350
+    _defaults['spec'] = 'auto'
```
Comment (MRitter95) [inline bondpad.py]: I'd suggest changing the name to "type" or something. "spec" means a very specific thing in the lab

File: cpw/CPWBend.py [hunk 2]
```diff
@@ -5,7 +5,14 @@
 import numpy as np
 
 class CPWBend(Component):
-    """A CPW bend"""
+    """
+    creates a CPW bend with pinw/gapw/radius
+            
+    turn_angle: turn_angle is in degrees, positive is CCW, negative is CW
+    polyarc: True/False, True draws CPWBend as a polyline, False as arcs and lines
```
Comment (MRitter95) [inline cpw/CPWBend.py:12]: do we ever use the non polyline version?

File: docs/add_component_outline.txt [hunk 3]
```diff
@@ -40,19 +41,19 @@ class ComponentName(Component):
...
-        s.drawing.append(sdxf.PolyLine(pts))
-	pts = [(x1,y1),(x2,y2),(x3,y3),(x4,y4)]
-	...
-        
-	### or use existing CPW elements to create component
-	CPWStraight(s,settings = {'length':300,
-				  'pinw':20,
-				  'gapw':8.372
-				  })	
+		pts = [(x1,y1),(x2,y2),(x3,y3),(x4,y4)]
```
Comment (MRitter95) [inline docs/add_component_outline.txt]: what's going on with the white space here? Looks like you added a tab to everything but I can't figure out why

File: docs/.ipynb_checkpoints/MaskMaker_overview-checkpoint.ipynb [hunk 4]
```diff
@@ -0,0 +1,808 @@
+{
```
Comment (MRitter95) [inline docs/.ipynb_checkpoints/MaskMaker_overview-checkpoint.ipynb:1]: we should add a line in the .gitignore to ignore these backup files so they don't clutter the commit history/ storage space

**Review by MRitter95 — COMMENTED** [review]
Looks pretty much good to me. Left a few comments for you to update/ clarify

File: cpw/CPWBend.py (hunk 2 above)
Comment (theomgifford) [reply cpw/CPWBend.py:12]: I specifically removed it this version, but kept it as a setting to "not mess with existing files" (I don't think its ever been used with the current MaskMaker). I can just take it out completely, but it seemed more correct to keep it compatible with designs from previous versions.

File: docs/add_component_outline.txt (hunk 3 above)
Comment (theomgifford) [reply docs/add_component_outline.txt]: I probably got confused, I'll fix that.

File: cpw/CPWBend.py (hunk 2 above)
Comment (theomgifford) [reply cpw/CPWBend.py:12]: next time

**Pull request merged**